Dockerfile

```diff
@@ -6,7 +6,7 @@
 ##    docker build --no-cache -t vela-npm:local .    ##
 ###########################################################################
 
-FROM node@sha256:0e071f3c5c84cffa6b1035023e1956cf28d48f4b36e229cef328772da81ec0c5
+FROM node@sha256:fc6bab09ee48632520dfa9d044d3fa10fc62969b4758a2d63b9db8c898f310b5
```

Comment:
any appetite to pin this to a human readable version plus the hash instead? otherwise, this is the same as `:latest` right?

Reply:
Yeah, agreed. I'll put a PR to add a `lts-gallium`